lightning/src/ln/channelmanager.rs

```diff
@@ -13232,8 +13244,8 @@ where
 						// Our channel information is out of sync with the `ChannelMonitor`, so
 						// force the update to use the `ChannelMonitor`'s update_id for the close
 						// update.
-						let latest_update_id = monitor.get_latest_update_id();
-						update.update_id = latest_update_id.saturating_add(1);
+						let latest_update_id = monitor.get_latest_update_id().saturating_add(1);
+						update.update_id = latest_update_id;
```

Comment:
Was this behavior wrong previously? (i.e., using the previous `latest_update_id` below)